source/documentation/upgrade-guide/chap-Updates_between_Minor_Releases.html.md

```diff
@@ -8,6 +8,8 @@ title: Updates Between Minor Releases
 
 Updates to the oVirt Engine are released via the oVirt Project.
 
+Note: If you are using a [Self-Hosted Engine](/self-hosted/Self-Hosted_Engine_Guide/) this does not apply.
```

Comment:
You probably intended to add this line to inside the next section "Updating oVirt Engine", no?

But I wonder if we really need a different, separate procedure.